troubadix/plugins/script_version_and_last_modification_tags.py

```diff
@@ -94,6 +94,17 @@ def check_content(
                 file=nasl_file,
                 plugin=self.name,
             )
+        else:
+            version_str = version_match.group("value")
+            format_version = "%Y-%m-%dT%H:%M:%S%z"
```

Comment:
```suggestion
                plugin=self.name,
            )
            return

        version_str = version_match.group("value")
        format_version = "%Y-%m-%dT%H:%M:%S%z"
```
Maybe we can return early to avoid some indentation / one level of nesting (`yield` in `except` in `else`)

Reply:
This would skip the last_modification checks it currently performs, since it has already determined that the version is missing. The existing tests required it to check both in the same run, even if version failed.

Reply:
Ah, true.